crypto/err/err.c

```diff
@@ -807,6 +839,9 @@ int ERR_get_next_error_library(void)
     ret = int_err_library_number++;
     CRYPTO_THREAD_unlock(err_string_lock);
     return ret;
+#else
+    return int_err_library_number;
+#endif
```

Comment:
I disagree with the change in this function.  Discussion happening in the other PR: https://github.com/openssl/openssl/pull/17974#discussion_r840280039

Reply:
I just dropped the last commit 8a837406b88b3d6a7a441eda5835cbacde5426ac.